ts/boot/configureStoreAndPersistor.ts

```diff
@@ -324,7 +324,8 @@ const rootPersistConfig: PersistConfig = {
     "payments",
     "content",
     "userMetadata",
-    "crossSessions"
+    "crossSessions",
+    "lollipop"
```

Comment:
Should we add a migration for lollipop?
```Typescript
  // Version 21
  // add lollipop
  "21": (state: PersistedState) => {
    return {
      ...state,
      lollipop: initialLollipopState
    };
  }
```
We also need to increment the value of `CURRENT_REDUX_STORE_VERSION`

Reply:
I guess you're right. Even if we add something (without modifying the structure), this makes us robust on future migration that could change the `LollipopState` structure.